lib/src/cli/flutter_cli.dart

```diff
@@ -165,14 +167,15 @@ class Flutter {
         void noop(String? _) {}
         final target = DirectoryGeneratorTarget(Directory(p.normalize(cwd)));
         final workingDirectory = target.dir.absolute.path;
+        final relativePath = p.relative(workingDirectory, from: initialCwd);
```

Comment:
**Nit:**

Introduce a variable `relativePathPrefix` and use it to avoid duplicating the logic for the prefix on both calls.

```suggestion
        final relativePath = p.relative(workingDirectory, from: initialCwd);
        final relativePathPrefix = .${p.context.separator};
```

When doing so we should probably refactor the tests to include a method or variable, for example something as:
```dart
final relativePathPrefix = '.${p.context.separator}';
expect(
  stdoutLogs,
  equals([
    'Running "flutter test" in ${relativePathPrefix}foo ...\n',
```